charts/px-backup/templates/px-central-ui/pxcentral-ui-configmap.yaml

```diff
@@ -14,7 +15,11 @@ data:
   DB_PORT: "3306"
   DB_USERNAME: root
   DB_DATABASE: pxcentral
+  {{- if eq $operatorToChartUpgrade true }}
+  DB_PASSWORD: singapore
+  {{- else }}
   DB_PASSWORD: Password1
```

Comment:
We should store this in a secret and allow users to set this

Reply:
Looks like this is already in the secret for mysql, the backend component should read it from there

Reply:
Yes, In MYSQL statefulset I am reading password of a secret but for UI backend Binu needs it in configmap. so once Binu adds to change for reading the password from secret I can remove password from configmap.